docs/usage/self-hosted-experimental.md

```diff
@@ -134,6 +134,15 @@ Skip initializing `RE2` for regular expressions and instead use Node-native `Reg
 ## `RENOVATE_X_MERGE_CONFIDENCE_API_BASE_URL`
 
 If set, Renovate will query this API for Merge Confidence data.
+
+If you use Mend Renovate Enterprise and:
```

Comment:
```suggestion
If you use the Mend Renovate Enterprise Edition (Renovate EE) and:
```

I think `Mend Renovate Enterprise Edition` is the preferred branding of the paid offering? It's what they use on [the Mend.io site, Mend Renovate Enterprise Edition page](https://www.mend.io/renovate-enterprise/).

Also, here's a quote from the [Renovate docs, running Renovate](https://docs.renovatebot.com/getting-started/running/#mend-renovate-community-edition-enterprise-edition):

> Mend Renovate Community Edition (Renovate CE) and Enterprise Edition (Renovate EE) are closed-source offerings of Renovate for self-hosted users.

Reply:
Nice clarification!